src/runtime/ai/server/utils/ai.ts

```diff
@@ -63,6 +64,42 @@ export function hubAI(): Ai {
   }
   if (!_ai) {
     throw createError('Missing Cloudflare AI binding (AI)')
+    const cfAccessHeaders = getCloudflareAccessHeaders(hub.cloudflareAccess)
+    _ai = proxyHubAI(hub.projectUrl, hub.projectSecretKey || hub.userToken, cfAccessHeaders)
+    return _ai
+  }
+  if (binding) {
+    if (import.meta.dev) {
+      // Mock _ai to call NuxtHub Admin API to proxy CF account & API token
+      _ai = {
+        async run(model: string, params?: Record<string, unknown>) {
+          if (!hub.projectKey) {
+            throw createError({
+              statusCode: 500,
+              message: 'Missing hub.projectKey variable to use hubAI()'
+            })
+          }
+          if (!hub.userToken) {
+            throw createError({
+              statusCode: 500,
+              message: 'Missing hub.userToken variable to use hubAI()'
+            })
+          }
+          return $fetch(`/api/projects/${hub.projectKey}/ai/run`, {
+            baseURL: hub.url,
+            method: 'POST',
+            headers: {
+              authorization: `Bearer ${hub.userToken}`
+            },
+            body: { model, params },
+            responseType: params?.stream ? 'stream' : undefined
+          }).catch(handleProxyError)
+        }
+      } as Ai
+    } else {
+      _ai = binding as Ai
+    }
+    return _ai
```

Comment:
Are you sure about those changes here?

As we throw the error line 66.

I gues you want to get the headers on between line 33 and 34 right?

Reply:
I think the merging main went wrong 😄 Yeah, it'll just be those two lines